zrml/prediction-markets/src/lib.rs

```diff
@@ -947,6 +947,10 @@ mod pallet {
         #[pallet::constant]
         type AdvisoryBond: Get<BalanceOf<Self>>;
 
+        /// The percentage of the advisory bond that gets slashed when a market is rejected.
+        #[pallet::constant]
+        type AdvisoryBondSlashPercentage: Get<Percent>;
```

Comment:
In `admin_destroy_market` it's okay to slash `100%`? I am fine with this, because sudo action should be heavy.

Reply:
Yes, I think so. There's no point in calling `admin_destroy_market` on a market that's proposed because you can achieve the same result by rejecting the market. And when the market is no longer proposed, then the `AdvisoryBond` has already been released.